osf/utils/datetime_aware_jsonfield.py

```diff
@@ -116,6 +119,7 @@ def prepare_value(self, value):
                 params={'value': value},
             )
```

Comment:
Ditto. `django.forms.JSONField ` (imported as `JSONFormField`) has a `to_pyton()` that we can call before adding our customization.